cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -214,6 +215,49 @@ def test_embedded_ansible_repository_invalid_url_crud(request, appliance, wait_f
     repository.delete_if_exists()
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_add_private_repository_crud(
+    request, credentials_collection, appliance, wait_for_ansible
+):
+    """
+    Add Private Repository by using SCM credentials.
```

Comment:
Please add more info about the test case like `Repository status should show failed or error on adding private repo`??

Reply:
Updated information